backend/src/server.ts

```diff
@@ -1,17 +1,102 @@
 import express from 'express';
+import WebSocket from 'ws';
+import * as fs from 'fs';
+
 import * as code from './code';
+import { handleConnection } from './connection';
 
 const app = express();
-const used: string[] = [];
+const usedGame: string[] = [];
+// used ids for both players and host
+const usedId: string[] = [];
+// first key is gameId, second key is attributes (data, host)
+const games: Map<string, Map<string, string>> = new Map();
+
 
 app.get('/', (_req, res) => {
 	res.send('Hello, world!');
 });
 
-app.get('/code', (_req, res) => {
-	res.send(code.gen(5, used));
+app.post('/games', express.json(), (req, res) => {
+	const file = req.body;
+	try {
+		const data = fs.readFileSync(file, 'utf8');
```

Comment:
What I intended in the schema is that `req.body` is already the whole JSON object, not the path to a file on the server or something Sorry if that was unclear. Like:

```
POST /games HTTP/1.1
Content-Length: 326
Content-Type: application/json

{"questions":[{"questionText":"questions can come in any order","answerTexts":["Order of questions is specified in the `ordering` field","Which is required","So they can appear in any order in the json","But Please have the questions appear in the correct order in the json","Ordering starts at zero"],"correctAnswers":[0]}]}
```

So I think this will just be `const data = JSON.parse(req.body)`.